src/components/ArticleHeadlines.js

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom'
```

Comment:
I think you're missing a ; at the end?

import { Link, useParams } from 'react-router-dom';